cfme/exceptions.py

```diff
@@ -285,3 +285,10 @@ class PaginatorException(CFMEException):
     """Raised by functions in :py:mod:`cfme.web_ui.paginator`"""
 
     pass
+
+
+class MWServerNotFound(CFMEException):
```

Comment:
I guess the contraction here is obscuring the exception purpose. MiddlewareServerNotFound is ok to be used.